src/views/Verify.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div>
+    <h1>You have to verify your email!</h1>
+    <button @click="singOut">SING OUT</button>
```

Comment:
These texts should be in the i18n files :D!

Reply:
These texts will be removed when I start to work on the dashboard! :d It's just for test